requirements.txt

```diff
@@ -0,0 +1,5 @@
+pymodbus
```

Comment:
not a blocker in this PR, but I feel we should we following the common migration towards poetry. https://evergenworkspace.slack.com/archives/CJE3YMPU7/p1684905901954769

Reply:
Feel free to address that as a separate piece of work